classes/blocks/class-accordion.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Accordion block class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Blocks;
+
+/**
+ * Class Accordion
+ *
+ * @package P4GBKS\Blocks
+ */
+class Accordion extends Base_Block {
+
+	/**
+	 * Block name.
+	 *
+	 * @const string BLOCK_NAME.
+	 */
+	const BLOCK_NAME = 'accordion';
+	const MAX_ROWS   = 5;
+
+	/**
+	 * Accordion constructor.
+	 */
+	public function __construct() {
+		register_block_type(
+			'planet4-blocks-beta/accordion',
+			[
+				'editor_script'   => 'planet4-blocks-beta',
+				'render_callback' => [ $this, 'render' ],
```

Comment:
I don't think that we need this `render_callback`, since this is a brand new block 🤔